File: Easy/lc58.py
# Length of last word
class Solution(object):
    def lengthOfLastWord(self, s):
        y=s.split()
        return len(y[-1])
s=Solution()
print(s.lengthOfLastWord("Hello World"))  # Output: 5
print(s.lengthOfLastWord("   fly me   to   the moon  "))  # Output: 4
print(s.lengthOfLastWord("luffy is still joyboy"))  # Output: 6

File: Easy/lc58.java
class Solution {
    public int lengthOfLastWord(String s) {
        s=s.trim();
        int count=0;
        int i=s.length()-1;
        while(i>=0 && s.charAt(i)!=' '){
            count++;
            i--;
        }
        return count;
    }
    public static void main(String[] args) {
        Solution sol = new Solution();
        String s = "Hello World";
        System.out.println(sol.lengthOfLastWord(s)); // Output: 5
    }
}